ui/app/mainui/Popups.qml

```diff
@@ -933,7 +943,67 @@ QtObject {
 
                 onDeclineClicked: root.ownershipDeclined()
             }
-        }
+        },
         // End of components related to transfer community ownership flow.
+        Component {
+            id: confirmHideAssetPopup
+            ConfirmationDialog {
+
+                property string assetSymbol
+                property string assetName
+                property string assetImage
+
+                width: 520
+                destroyOnClose: true
+                confirmButtonLabel: qsTr("Hide asset")
+                cancelBtnType: ""
+                showCancelButton: true
+                headerSettings.title: qsTr("Hide %1 (%2)").arg(assetName).arg(assetSymbol)
+                headerSettings.asset.name: assetImage
+                confirmationText: qsTr("Are you sure you want to hide %1 (%2)? You will no longer see or be able to interact with this asset anywhere inside Status.").arg(assetName).arg(assetSymbol)
+                onCancelButtonClicked: close()
+                onConfirmButtonClicked: {
+                    root.walletAssetsStore.manageAssetsController.settingsHideToken(assetSymbol)
+                    close()
+                    Global.displayToastMessage(qsTr("%1 (%2) successfully hidden. You can toggle asset visibility via %2.").arg(assetName).arg(assetSymbol)
+                                               .arg(`<a style="text-decoration:none" href="#${Constants.appSection.profile}/${Constants.settingsSubsection.wallet}/${Constants.walletSettingsSubsection.manageAssets}">` + qsTr("Settings", "Go to Settings") + "</a>"),
+                                               "",
+                                               "checkmark-circle",
+                                               false,
+                                               Constants.ephemeralNotificationType.success,
+                                               "")
+                }
+            }
+        },
+        Component {
+            id: confirmHideCollectiblePopup
+            ConfirmationDialog {
+
+                property string collectibleSymbol
+                property string collectibleName
+                property string collectibleImage
+
+                width: 520
+                destroyOnClose: true
+                confirmButtonLabel: qsTr("Hide collectible")
+                cancelBtnType: ""
+                showCancelButton: true
+                headerSettings.title: qsTr("Hide %1").arg(collectibleName)
+                headerSettings.asset.name: collectibleImage
+                confirmationText: qsTr("Are you sure you want to hide %1? You will no longer see or be able to interact with this collectible anywhere inside Status.").arg(collectibleName)
+                onCancelButtonClicked: close()
+                onConfirmButtonClicked: {
+                    root.walletAssetsStore.manageCollectiblesController.settingsHideToken(collectibleSymbol)
```

Comment:
This is not the right controller, it lives in `AppMain.qml` under `readonly property WalletStore.CollectiblesStore walletCollectiblesStore: WalletStore.RootStore.collectiblesStore` so you also have to pass it down correctly to `Popups.qml`

Reply:
Ups, wrong spelling yes!